package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Listen to YouTube videos, without the distracting visuals.",
   "main": "tests.js",
   "scripts": {
-    "pretest": "jshint js test",
+    "lint:js": "jshint js test",
+    "lint:html": "htmlhint index.html",
+    "pretest": "npm run lint:js && npm run lint:html",
```

Comment:
I like the way you broke this up into multiple scripts. For `pretest`, can we do something like `npm run test -- --grep="lint"`. It seems possible as of npm v2.0.0